Zuul-MSSC/src/main/resources/application.properties

```diff
@@ -22,7 +22,9 @@ zuul.routes.auth.strip-prefix=false
 zuul.routes.auth.sensitive-Headers=Cookie,Set-Cookie
 zuul.sensitive-headers=Cookie,Set-Cookie
 
-
+# On slow environments , there are annoying ZUUL timeouts
```

Comment:
These properties solve the problem of long runtime of the whole system?

Reply:
This solves the problem when the calls to services and from services to DBs take longer than ZUUL can stand. Timeouts happen on gateway quite often on my machine without it

Reply:
ok

Reply:
ok